ui/app/routes/vault/cluster/secrets/backend/configuration/index.js

```diff
@@ -149,6 +129,49 @@ export default class SecretsBackendConfigurationRoute extends Route {
     }
   }
 
+  async fetchGcpConfig(id) {
+    try {
+      const gcpModel = await this.store.queryRecord('gcp/config', { backend: id });
+      let issuer = null;
+      if (this.version.isEnterprise && gcpModel) {
```

Comment:
Suggestion based on comment above, since we'll hit the catch block if the request fails so I don't thin the second check here is necessary
```suggestion
      if (this.version.isEnterprise) {
```